test/e2e/persistent_volumes.go

```diff
@@ -272,6 +293,45 @@ func waitAndValidatePVandPVC(c *client.Client, ns string, pv *api.PersistentVolu
 	return pv, pvc, nil
 }
 
+// Validates PVs and PVCs for phase "Bound".  Once all pairs are validated, updates pvList & pvcList data
+// Takes a list of PVs and a list of PVCs
+// Returns a list of PVs, a list of PVCs, and nil (or error)
+func waitAndValidateMultiplePVandPVCPairs(c *client.Client, ns string, pvList []*api.PersistentVolume, pvcList []*api.PersistentVolumeClaim) ([]*api.PersistentVolume, []*api.PersistentVolumeClaim, error) {
```

Comment:
If the pvList is paired with the pvcList, meaning that for all pvs, pvList[i] is bound to pvcList[i], then this func could be a wrapper for waitAndValidateMultiplePVandPVC(). You would get the LogF and reuse code.